_aicspylibczi/Reader.h

```diff
@@ -277,25 +279,28 @@ namespace pylibczi {
        */
       libCZI::IntRect getSceneYXSize(int scene_index_ = -1);
 
-      std::string pixelType() const {
+      std::string pixelType() {
           // each subblock can apparently have a different pixelType 🙄
+          if(m_pixelType == libCZI::PixelType::Invalid) m_pixelType = getFirstPixelType();
```

Comment:
could this be initialized earlier in order to keep the method const?

Reply:
I'm essentially lazy loading here, I pushed things like this off so that we weren't taking performance hits for all the "we should initialize these up front". I have to have a think on this.